README.md

```diff
@@ -160,19 +160,21 @@ Supports the [TeamCity](https://www.jetbrains.com/teamcity/) build service.
     "branch": "master",
     "authentication": "ntlm",
     "username": "teamcity_username",
-    "password": "teamcity_password"
+    "password": "teamcity_password",
+    "useGuest": true
   }
 }
 ```
 
 | Setting                 | Description
 |-------------------------|-----------------------------------------------------------------------------------------
-| `url`                   | The url to the TeamCity server (including the credentials without a trailing backslash).
+| `url`                   | The url to the TeamCity server (including the credentials without a trailing backslash, if not the guest user is used).
```

Comment:
Is that comment really needed? Does it make a difference if `/guestAuth` or `/httpAuth` is included in the url?

Reply:
Ah, forget about it. It doesn't have to do anything about the url formatting. ;-)